### DISCUSSION TITLE
[2679460]Metaphor: ReFantazio

### DISCUSSION THREAD

**Discussion by RikuKH3** [pjy612/SteamManifestCache#886] [Share]
### Steam 游戏ID

2679460

### 密钥列表

```hash
2679461;0a945c91ddfaaca28d19da03981f783f8eb23a6dcc1121fb8c0936bb17fe0adb
```

### 清单附件

[2679461_8392723469355125002.zip](https://github.com/user-attachments/files/17495878/2679461_8392723469355125002.zip)

**Comment by pjy612** [comment]
w(ﾟДﾟ)w, Is you the one who cracked the Metaphor: ReFantazio? ! ? !

**Reply by RikuKH3** [reply]
Yeah, I added missing stubbed functions from denuvo protected executable into the demo.

**Reply by pjy612** [reply]
Worship the boss up close ...  ∩(。_。)∩

**Reply by ZEERDEER** [reply]
🙆

**Reply by fatelove42** [reply]
🙌
